languagetool-language-modules/de/src/main/java/org/languagetool/tokenizers/de/GermanCompoundTokenizer.java

```diff
@@ -175,8 +175,32 @@ public GermanCompoundTokenizer(boolean strictMode) throws IOException {
     // add exceptions here so we don't need to update JWordSplitter for every exception we find:
     //wordSplitter.addException("Maskerade", Collections.singletonList("Maskerade"));
     //wordSplitter.addException("Sportshorts", asList("Sport", "shorts"));
+    wordSplitter.addException("Absolventen", asList("Absolventen"));
+    wordSplitter.addException("Acetat", asList("Acetat"));
     wordSplitter.addException("Alkoholabstinenz", asList("Alkohol", "abstinenz"));
+    wordSplitter.addException("Androgen", asList("Androgen"));
+    wordSplitter.addException("Auberginen", asList("Auberginen"));
+    wordSplitter.addException("Broadcast", asList("Broadcast"));
+    wordSplitter.addException("Buchsbaum", asList("Buchsbaum"));
+    wordSplitter.addException("Coffein", asList("Coffein"));
+    wordSplitter.addException("Drohnen", asList("Drohnen"));
+    wordSplitter.addException("Eiben", asList("Eiben"));
+    wordSplitter.addException("Eingroschen", asList("Eingroschen"));
+    wordSplitter.addException("Einkomponenten", asList("Einkomponenten"));
+    wordSplitter.addException("Elster", asList("Elster"));
+    wordSplitter.addException("Engineering", asList("Engineering"));
+    wordSplitter.addException("Factoring", asList("Factoring"));
+    wordSplitter.addException("Flexions", asList("Flexions"));
+    wordSplitter.addException("Flexodruck", asList("Flexo", "druck"));
+    wordSplitter.addException("Graviton", asList("Graviton"));
+    wordSplitter.addException("Göttinnen", asList("Göttinnen"));
     wordSplitter.addException("Hallesche", asList("Hallesche"));
+    wordSplitter.addException("Hinspiel", asList("Hinspiel"));
+    wordSplitter.addException("Homogen", asList("Homogen"));
+    wordSplitter.addException("Infraschall", asList("Infraschall"));
+    wordSplitter.addException("Innovations", asList("Innovations"));
+    //wordSplitter.addException("Induktivitäts", asList("Induktivitäts"));
+    wordSplitter.addException("Jahreszeiten", asList("Jahreszeiten"));
```

Comment:
Gibt es einen Grund, das nicht in Jahres+zeiten zu splitten?

Reply:
Ja, weil es Komposita gibt, in denen 'zeit' verlangt wird und nicht 'zeiten' (z. B. Echtzeitüberweisung)